### DISCUSSION TITLE
可以按月或是按季度或是按年来存档文件么？

### DISCUSSION THREAD

**Discussion by Wattskemov** [Quorafind/Obsidian-Thino#299] [Ideas 想法]
首先这个插件很好用，但是就是用daily notes的话，一天一个文件，导致文件太多了。

有没有可能按一个月或是一个季度或是一年来收集信息？

我试了一下 Periodic，把memos 设置为使用 Pedriodic，Pedriodic设置为按季度收集信息，但是好像没有效果。

**Comment by Quorafind** [comment]
重构后会解决这个问题，这次重构解决的问题主要是缓存数据、获取数据的问题，参考这里的图 https://github.com/Quorafind/Obsidian-Memos/pull/282#issuecomment-1374860737

**Reply by Wattskemov** [reply]
期待，重构的版本啥时候出？

**Reply by Quorafind** [reply]
不承诺时间，每天干多少是多少
